content/docs/1.5.0/references/networking.md

```diff
@@ -45,16 +46,16 @@ To | Port | Protocol
 #### ingress
 From | Port | Protocol
 --- | --- | ---
-`Longhorn Manager` | 8500; 10000-30000 | TCP
+`Longhorn Manager` | 8500; 8501 | TCP
 `Other Instance Manager` | 10000-30000 | TCP
 `Node in the Cluster` | 3260 | TCP
-`Backing Image Manager` | 10000-30000 | TCP
+`Backing Image Data Source` | 10000-30000 | TCP
```

Comment:
I forgot why instance-manager needs to talk to the backing image manager. Is it for exporting volume?

Reply:
I think so. I didn't find any other case. 
And now volume export is handled by Backing Image Data Source instead.

Reply:
> start a replica client to ask instance manger to export the volume

This means that the BIDS pod will send a file sync request to the replica process inside the instance manager pod:
https://github.com/longhorn/longhorn-manager/blob/b9ff9347ed259338340211a62730e6c3261e42bb/controller/backing_image_data_source_controller.go#L765
Here, the port of the replica address is range 10000-30000.